setup.py

```diff
@@ -142,7 +142,6 @@ def readfile(fname):
         'Hierarchical Dirichlet Process, HDP, Random Projections, '
         'TFIDF, word2vec',
 
-    license='LGPL',
```

Comment:
I was trying to put v2.1 everywhere. Please let me know if there is v2 that has been accidentally added anywhere.

Do you mean that the PyPi classifier refers to v2?

Unfortunately there is no LGPL v2.1  [PyPi classifier](https://pypi.python.org/pypi?%3Aaction=list_classifiers) .
LGPL v2.1 falls under the LGPLv2+ that we use 'License :: OSI Approved :: GNU Lesser General Public License v2 or later (LGPLv2+)'. We are explicit in COPYING that it is specifically v2.1 so it should be ok.